window/WindowAsAService/Pages/Index.razor

```diff
@@ -0,0 +1,52 @@
+﻿@page "/"
+@using WindowAsAService.Models
+@using WindowAsAService.Services
+
+<h1>Hello, world!</h1>
+
+Welcome to your new app.
+<br />
+
+WindowTitle:
+<TelerikTextBox @bind-Value="@WindowTitle"></TelerikTextBox>
+<br />
+
+WindowContent
+<TelerikTextBox @bind-Value="@WindowContent"></TelerikTextBox>
```

Comment:
Consider a `TelerikEditor` here, combined with a `MarkupString` inside `WindowContent`. This will allow richer content and more flexible example.

Reply:
We chose to leave a textbox as there are many options to set the content of the Window. It could be a plain string, markup, or an inline render fragment. I think the clients must decide how to define that and we just show the simplest representation. If you think that an Editor with a MarkupString will bring the additional value to the clients we can easily add.